package.json

```diff
@@ -72,7 +73,7 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "^29.5.0",
     "jest-environment-jsdom": "^29.5.0",
-    "jest-environment-jsdom-fifteen": "^1.0.2",
+    "msw": "1.3.2",
```

Comment:
I'm going to leave this here as I am about to put another PR up which adds MSW. I wanted to do them in separate PRs rather than make a giant one. It also is useful to validate all the tests work correctly as they are now before utilizing MSW.